CODEOWNERS

```diff
@@ -360,7 +365,9 @@ jenkinsfiles @cilium/ci-structure
 /pkg/counter @cilium/sig-datapath
 /pkg/datapath/ @cilium/sig-datapath
 /pkg/datapath/linux/config/ @cilium/loader
+/pkg/datapath/linux/ipsec/ @cilium/ipsec
 /pkg/datapath/linux/ipsec/xfrm_collector* @cilium/metrics
+/pkg/datapath/linux/node.go @cilium/ipsec @cilium/sig-datapath
```

Comment:
I'm OK with this for now. Over time we should restructure this code so that encryption pieces can just subscribe to the events they're looking for from the node event and avoid having to review the core code here. Maybe by pulling you folks in more often, it'll help motivate addressing that tech debt ;)

Reply:
Yep, agree. I'm refraining from doing too much refactoring for now, but we should be able to easily extract the IPsec code from that file.